src/Http/Http.Features/src/PublicAPI.Shipped.txt

```diff
@@ -61,9 +61,6 @@ Microsoft.AspNetCore.Http.Features.IFormFeature.ReadFormAsync(System.Threading.C
 Microsoft.AspNetCore.Http.Features.IHttpBodyControlFeature
 Microsoft.AspNetCore.Http.Features.IHttpBodyControlFeature.AllowSynchronousIO.get -> bool
 Microsoft.AspNetCore.Http.Features.IHttpBodyControlFeature.AllowSynchronousIO.set -> void
-Microsoft.AspNetCore.Http.Features.IHttpBufferingFeature
```

Comment:
Are there no concrete implementations for these feature?

Reply:
No, they were replaced in 3.0 by IHttpResponseBodyFeature.